faucet/cmd/health/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"context"
+	"expvar"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/ardanlabs/conf/v3"
+	"github.com/gorilla/handlers"
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+
+	app "github.com/filecoin-project/faucet/internal/http"
+	"github.com/filecoin-project/faucet/internal/platform/lotus"
+	"github.com/filecoin-project/lotus/api/client"
+)
+
+var build = "develop"
+
+func main() {
+	logger := logging.Logger("SPACENET-HEALTH")
+
+	lvl, err := logging.LevelFromString("info")
+	if err != nil {
+		panic(err)
+	}
+	logging.SetAllLoggers(lvl)
+
+	if err := run(logger); err != nil {
+		logger.Fatalln("main: error:", err)
+	}
+}
+
+func run(log *logging.ZapEventLogger) error {
+	// =========================================================================
+	// Configuration
+
+	cfg := struct {
+		conf.Version
+		Web struct {
+			ReadTimeout     time.Duration `conf:"default:5s"`
+			WriteTimeout    time.Duration `conf:"default:60s"`
+			IdleTimeout     time.Duration `conf:"default:120s"`
+			ShutdownTimeout time.Duration `conf:"default:20s"`
+			Host            string        `conf:"default:0.0.0.0:9000"`
+		}
+		TLS struct {
+			Disable  bool   `conf:"default:true"`
+			CertFile string `conf:"default:nocert.pem"`
+			KeyFile  string `conf:"default:nokey.pem"`
+		}
+		Lotus struct {
+			APIHost   string `conf:"default:127.0.0.1:1230"`
+			AuthToken string
+		}
+	}{
+		Version: conf.Version{
+			Build: build,
+			Desc:  "Spacenet Health Service",
+		},
+	}
+
+	const prefix = "HEALTH"
```

Comment:
Why is the prefix needed?

Reply:
Just for logging in cmd binary. But I removed it